pkg/crc/machine/bundle/metadata.go

```diff
@@ -250,10 +251,21 @@ func GetCustomBundleName(bundleFilename string) string {
 	return fmt.Sprintf("%s_%d%s", baseName, time.Now().Unix(), bundleExtension)
 }
 
-type presetDownloadInfo map[preset.Preset]*download.RemoteFile
+func GetBundleNameFromURI(bundleURI string) string {
+	// the URI is expected to have been validated by validation.ValidateBundlePath first
+	if strings.HasPrefix(bundleURI, "docker://") {
+		imageAndTag := strings.Split(filepath.Base(bundleURI), ":")
+		return constants.BundleForPreset(image.GetPresetName(imageAndTag[0]), imageAndTag[1])
+	}
```

Comment:
the `strings.Split` might deserver a helper in `image`, but that can be done next time someone touches this code.